{
  new AssertThrows(IllegalArgumentException.class){
    public void test() throws Exception {
      new LocalConnectionFactoryBean().afterPropertiesSet();
    }
  }
.runTest();
}
